templates/applicants.phtml

```diff
@@ -29,6 +29,13 @@
                         <a href="#apprentice" aria-controls="apprentice" role="tab" data-toggle="tab">
                             Apprentice Applicants
                         </a>
+                    </li>
+                    <li class="pull-right">
+                        <label for="name">
+                            Search Name
+                            <input class="submitApplicant form-control" type="text" name="name" id="name" value=""/>
```

Comment:
does this need the id? is that for styling?

Reply:
this shouldn't even be in the repo! haha